torch/utils/data/_utils/worker.py

```diff
@@ -234,52 +268,21 @@ def _worker_loop(
     auto_collation,
     collate_fn,
     drop_last,
-    base_seed,
+    seed,
     init_fn,
     worker_id,
     num_workers,
     persistent_workers,
-    shared_seed,
+    shared_rng=None,
+    is_process=True,
+    watchdog_constructor=None,
+    error_prefix="worker process",
```

Comment:
Rather than adding an extra argument to this function, could do `error_prefix = "worker process" if is_process else "worker thread` inside the function body.